### ISSUE TITLE
Scans fail. Seems like an ascii bug: -&#45;script= 

### ISSUE THREAD

**Issue by healeyio** [Rev3rseSecurity/WebMap#63]
All scans exit as a success with 0 hosts up.
1. Nmap syntax: Any (from what I can tell). -sP even fails when global pings succeed.
2. WebMap in docker container: [x] Yes
3. WebMap version or branch: v2.3/master
3. Step to reproduce: Run a scan

**Expected behavior**
A scan.

**Screenshots**
![image](https://user-images.githubusercontent.com/9313140/50264651-6c813a00-03d0-11e9-8471-27fffe45275a.png)

**Additional context**
I'm pretty sure it's failing with the ascii code highlight in the screenshot. Here is how the nmap command shows up in the xml file output:
` nmap -sP -&#45;script=/opt/nmapdashboard/nmapreport/nmap/nse/ -oX /tmp/scannermcsanface.xml.active 8.8.8.8`

**Comment by theMiddleBlue** [comment]
Hi @healeyio 

thank you for reporting this. Is this happens when starting a new scan from the WebMap UI?

**Comment by healeyio** [comment]
Yes. It happens when I run a scan from the UI. I haven't tried via the api yet. I'll try to give that a shot today.
